pkgs/by-name/im/immich/package.nix

```diff
@@ -166,13 +166,6 @@ buildNpmPackage' {
     # see https://github.com/immich-app/immich/issues/13971
     substituteInPlace src/services/backup.service.ts \
       --replace-fail '`/usr/lib/postgresql/''${databaseMajorVersion}/bin/pg_dumpall`' '`pg_dump`'
-
-    # some part of the build wants to use un-prefixed binaries. let them.
-    mkdir -p $TMP/bin
-    ln -s "$(type -p ${stdenv.cc.targetPrefix}pkg-config)" $TMP/bin/pkg-config
-    ln -s "$(type -p ${stdenv.cc.targetPrefix}c++filt)" $TMP/bin/c++filt
-    ln -s "$(type -p ${stdenv.cc.targetPrefix}readelf)" $TMP/bin/readelf
-    export PATH="$TMP/bin:$PATH"
```

Comment:
These lines were for cross. I'll have to test if they're still required.

Reply:
I was able to build `pkgsCross.aarch64-multiplatform.immich.web` on my `x86_64-linux`